lib/cucumber/wire/protocol/requests.rb

```diff
@@ -64,6 +65,19 @@ def execute(step_definition_id, args)
             super(request_params)
           end
 
+          def handle_fail(params)
+            begin
```

Comment:
as the method signature is only begin/rescue/end, we can omit them (and the returns), so we get a slightly lighter

```
def handle_fail(params)
  raise ....
rescue Exception => exception
  Cucumber::Core.....
end
```

Reply:
Done.